kotlin/samples/hello-terminal/hello-terminal-app/src/main/java/com/squareup/sample/helloterminal/HelloTerminalWorkflow.kt

```diff
@@ -81,4 +75,14 @@ class HelloTerminalWorkflow : TerminalWorkflow,
   }
 
   override fun snapshotState(state: State): Snapshot = Snapshot.EMPTY
+
+  private fun onKeystroke(key: KeyStroke): HelloTerminalAction = WorkflowAction {
```

Comment:
This is the kind of place where I especially think `WorkflowAction {` carries its weight — these actions are all tied to worker calls, and so we never make a sink. An `enum` / `sealed class` would simply be boilerplate, and worse introduce an unused mechanism.

Reply:
+1 these are super clean